views/bigquery.go

```diff
@@ -180,9 +180,13 @@ func buildViewsEventsQuery(table string, spec QuerySpec) (string, []interface{},
 			OrderBy("time_interval")
 	}
 
-	if from := spec.From; from != nil {
-		query = query.Where("time >= timestamp_millis(?)", from.UnixMilli())
+	from := spec.From
+	if from == nil {
+		defaultFrom := time.Now().AddDate(0, 0, -7)
```

Comment:
I think there is a corner case here when `From` is not set but `To` is. We can either make this be "7 days before `To`" or only add the default when there is no `To`. WDYT?